server/runner/pom.xml

```diff
@@ -129,7 +129,7 @@
             <plugin>
                 <groupId>org.wildfly.plugins</groupId>
                 <artifactId>wildfly-maven-plugin</artifactId>
-                <version>3.0.0.Final</version>    
+                <version>2.1.0.Final</version>    
```

Comment:
You sure?

Reply:
Yes, it breaks the runner. Not too important, but dependably upgrade it a while ago, I just revert as it's not working